docs/whats_new.rst

```diff
@@ -150,8 +150,8 @@ Features
 - Implement :class:`junifer.markers.ReHoParcels` and
   :class:`junifer.markers.ReHoSpheres` markers by `Synchon Mandal`_ (:gh:`36`)
 
-- Implement :class:`junifer.markers.AmplitudeLowFrequencyFluctuationParcels` and
-  :class:`junifer.markers.AmplitudeLowFrequencyFluctuationSpheres` markers by
+- Implement ``junifer.markers.AmplitudeLowFrequencyFluctuationParcels`` and
```

Comment:
I would change this fragemtn to the right object: `.ALFFParcels` and `.ALFFSphere`